pattern-library/sass/components/_icons.scss

```diff
@@ -10,5 +10,362 @@
 .icon {
     display: inline-block;
     font-size: 1em;
-    fill: currentColor;
+    color: currentColor;
 }
+
+// ----------------------------
+// #ICONS
```

Comment:
I can explain more in detail, but our `_icons.scss` partial was pre-existing. I added icons generated from Icomoon to make it easier to manage. I left the `edx-icons.css` file because I was hesitant to modify the stuff from Icomoon, but... we really don't need the CSS file once we copy its contents here.